about.html

```diff
@@ -34,11 +34,16 @@

```

Comment:
Just out of curiosity, is there a reason why this is a single line?

Reply:
The header and footer are automatically created by a tool here at Microsoft - I just copied and pasted the source code from the tool, so I guess it just pasted in one line. :)

Reply:
We could format that area so it is easier to review in the future, assuming we make any changes to that. But since we probably won't change the header, we probably can leave it as is? Your thoughts @anawishnoff ? ^^

Reply:
Yeah this header is pretty finicky - to update it, you need to go through the tool and add new links/tabs. Since you have to go through that process to edit it anyways, I think it should be fine to keep it in this format.

Reply:
Okay, I see. Then let's leave it minified.